### ISSUE TITLE
Send to multiple users

### ISSUE THREAD

**Issue by zimt28** [Meteor-Reaction/Herald#60]
Is there a way to send a notification to a group of users? Or can I just loop through the users and set the notification for each? In my app, I'd like to send a notification to a group of people, and then delete it when one of them reads it or takes action, not sure yet.

**Comment by Kestanous** [comment]
Hmm ok let see if I understand. You want a notification that gets sent to a group of users but if *any* of them marks it as read (however that gets triggered) you want it marked as read for all?

**Comment by zimt28** [comment]
Exactly. So when there's a question, anyone gets notified and can answer. But when it has been answered, all the other people don't have to look at it anymore.

**Comment by Kestanous** [comment]
Currently no that is not possible, I need not consider that use case. Currently herald crates a separate notification for each user so it can be customized to that user and so we can keep read state separated. Obviously the later is counter to what you need. Let me think on this, I am ok with adding this functionality but I am not sure how just yet.

**Comment by delgermurun** [comment]
This is also needs me. :+1:
